EVENT.md

```diff
@@ -119,10 +119,11 @@ are two types of attribution sources, `navigation` sources and `event` sources.
 or via calls to `window.open` that occur with [transient
 activation](https://html.spec.whatwg.org/multipage/interaction.html#transient-activation):
 ```javascript
+const encoded = encodeURIComponent('https://adtech.example/attribution_source?my_ad_id=123');
```

Comment:
A one-line comment explaining why we need to do this would be good. Something like:
```
// Encode the attributionsrc URL in case it contains "=" which will cause the parameter to be improperly parsed
```